README.md

```diff
@@ -411,6 +411,12 @@ There are three things you would need to do this -
 
 Once we have these three things we can then use the push notifications tester and send a local push notification to make sure that everything was set up correctly.
 
+
+#### Sandbox Support
+
+Apple have two environments with push notification support - production, and Sandbox. Sandbox applications that support push notifications are those signed with iOS Development Certificates instead of iOS Distribution Certificates. Applications in the Sandbox environment act as a staging environment allowing you to test your applications in a environment close-to but distinct from production.
```

Comment:
have or has? 🤔

Reply:
"Have or has" is a longtime British English vs American English collective noun debate. In the US, we'd typically say "Apple has" as we think of it as one unit whereas in England/Ireland, they say "Apple have" because they think of it as a bunch of individual people working together. 

"For instance, in American English, collective nouns are considered singular (e.g. The band is playing). In contrast, collective nouns can be either singular or plural in British English, although the plural form is most often used (e.g. The band are playing)."

Personally, I would put "Apple has two environments."

Reply:
I like this, will update!

Reply:
I assume no comments on the second paragraph then?